pagopa/io-app
# chore: [IA-764] Premium Messages Opt-In/Out profile stub screen

## Short description
This PR includes the initial logic for the Premium Messages opt-in/out screen in the `Profile` tab. This screen will show up inside the `Privacy Policy and Terms of Service` section.

https://user-images.githubusercontent.com/5963683/161984725-13612b80-8caf-40db-b98c-3646e67b913f.mov

⚠️ The `PremiumMessagesOptInOutProfileScreen` screen is just a stub so the text is hard-coded and not translated.

## List of changes proposed in this pull request
- Created a stub `PremiumMessagesOptInOutProfileScreen`
- Created the navigation to access the `PremiumMessagesOptInOutProfileScreen`

## How to test
1. Enable `PREMIUM_MESSAGES_OPT_IN_ENABLED`
2. Enable the feature in the dev server `premiumMessages.opt_in_out_enabled`

Now inside the `Privacy Policy and Terms of Service` section in the `Profile` tab the new option `Premium Messages Opt-in/out` should be visible and navigable.
